node/standalone/src/chain_spec.rs

```diff
@@ -99,7 +100,7 @@ fn fetch_xdns_record_from_rpc(
                 genesis_hash: client.genesis_hash.0.to_vec(),
             },
             gateway_sys_props,
-            vec![],
+            vec![*b"tran"],
```

Comment:
is this change on purpose?

Reply:
ish, in preparation for the transfer demo. But I guess we have to register the gateway manually for now, so maybe it should go

Reply:
cool